server/api/users/users.controller.ts

```diff
@@ -19,43 +19,23 @@ export const getCurrentUser: RequestHandler<void, ClientUser>  = (req, res) => {
 };
 
 export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req, res) => {
-  const user = await User.findByPk(req.params.id, {
-    attributes: [
-      'id',
-      'discord_user_id',
-      'discord_name',
-      'bio',
-      'twitter_username',
-      'linkedin_url',
-      'github_username',
-      'website',
-    ],
-  });
+  const allowedFields = User.allowedFields;
+  const user = await User.findByPk(req.params.id)
 
   if (!user) {
     throw new NotFoundError('User not found');
   }
-
-  res.json(user);
+  
+  const pickedUser = _.pick(user, allowedFields) as UserProfile
+  res.json(pickedUser);
 };
 
 export const getUsers: RequestHandler<UserProfile[]> = async (req, res) => {
-  const users = await User.findAll({
-    attributes: [
-      'id',
-      'discord_user_id',
-      'discord_name',
-      'bio',
-      'twitter_username',
-      'linkedin_url',
-      'github_username',
-      'website',
-    ],
-  });
-
-  if (users.length === 0) {
-    throw new NotFoundError('No Users found');
-  }
+  const allowedFields = User.allowedFields;
+  const users = await User.findAll()
```

Comment:
Same as above... inline `allowedFields`. You can then delete lines 34, 39, and 3; then return `users` in the response.
```js
const users = await User.findAll({ attributes: allowedFields });
```